packages/components/src/lib/plotting.ts

```diff
@@ -0,0 +1,90 @@
+import { Distribution, result, squiggleExpression } from "@quri/squiggle-lang";
```

Comment:
I don't like the name of this file; plotting.ts. Can it be more specific? It seems to be ``plotParser`` or something